### ISSUE TITLE
[TalkBack] problem while reading very long text (detailed!)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#249]
```
What steps will reproduce the problem?
1. Make TalkBack focus on a very long text inside a TextView

What is the expected output? What do you see instead?
It should start reading the text. Nothing happens, I just see the text that 
should be read (I have that TalkBack option enabled).

What version of the product are you using? On what operating system?
Android 4.1.2 - Talkback @ Samsung Galaxy S3 in spanish.

ADDITIONAL INFORMATION:
- I'm using default talkback version that comes with Android 4.1.2 since the 
update ONLY reads english.. while this version works for both spanish and 
english.

- I'm attaching the layout that has the issue to give more insight into this. 
You can see TextView id=leyenda0 has a short text and it reads fine.. the next 
TextView id=leyenda00 is the long text that doesn't read.

Thanks for your help!!
```

Original issue reported on code.google.com by `smur...@gmail.com` on 24 Apr 2013 at 7:46

Attachments:
- [leyendas_grales.xml](https://storage.googleapis.com/google-code-attachments/eyes-free/issue-249/comment-0/leyendas_grales.xml)

**Comment by GoogleCodeExporter** [comment]
```
Please update to the latest Play Store version of TalkBack. If this issue 
persists, please reopen this bug.
```

Original comment by `alanv@google.com` on 30 Apr 2013 at 7:46
- Changed state: **Invalid**
